Calculator.Console/Program.cs

```diff
@@ -6,8 +6,48 @@ class Program
     {
         static void Main(string[] args)
         {
-            System.Console.WriteLine("Hello World!");
-            var name = System.Console.ReadLine();
+            int a = 0, b = 0;
+            int result = 0;
+            char operationChoice = '\0';
+
+            Menu(ref operationChoice);
```

Comment:
Avoid using ref. Instead you can return.
Menu is not a good name for a function. Find a good one :)